timesketch/lib/tasks.py

```diff
@@ -115,21 +119,20 @@ def _get_index_analyzers():
         None if index analyzers are disabled in config.
     """
     tasks = []
+    index_analyzers = current_app.config.get('AUTO_INDEX_ANALYZERS')
 
-    # Exit early if index analyzers are disabled.
-    if not current_app.config.get('ENABLE_INDEX_ANALYZERS'):
+    if not index_analyzers:
         return None
 
-    for analyzer_name, analyzer_cls in manager.AnalysisManager.get_analyzers():
+    for analyzer_name, analyzer_cls in manager.AnalysisManager.get_analyzers(
+            index_analyzers):
         kwarg_list = analyzer_cls.get_kwargs()
```

Comment:
wasn't get_kwargs() removed from the interface?

Reply:
Fixed.